crates/anvil/core/src/eth/transaction/mod.rs

```diff
@@ -39,11 +42,14 @@ pub fn transaction_request_to_typed(tx: TransactionRequest) -> Option<TypedTrans
         gas_price,
         max_fee_per_gas,
         max_priority_fee_per_gas,
+        max_fee_per_blob_gas,
```

Comment:
I forget, why don't we do this with a `From` impl / reuse any alloy types code? Is it due to the Deposit tx I guess?

Reply:
nit there's a `Some(126)` below i'd like to replace it with a const that indicates the semantic value of the number from op specs